assets/js/components/dashboard-sharing/DashboardSharingSettings/Module.js

```diff
@@ -39,10 +39,9 @@ import {
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { Tooltip } from 'googlesitekit-components';
+import { Tooltip, Select } from 'googlesitekit-components';
```

Comment:
This is not a big deal, but should we sort the imports here? ie. `googlesitekit-components` before `googlesitekit-data` or `Select. Tooltip` instead of `Tooltip, Select`. Maybe this is something we should enforce project-wide and fix in a new issue using something like [eslint-sort-imports](https://eslint.org/docs/latest/rules/sort-imports)

Reply:
Updated all the imports in this PR, also opened #7275 to enforce it project-wide. Thank you!

Reply:
Thank you for updating the imports here and creating the issue for fixing this project-wide!